PhysicsTools/Heppy/python/analyzers/core/L1TriggerAnalyzer.py

```diff
@@ -0,0 +1,94 @@
+"""
+L1TriggerAnalyzer is an Analyzer that load in-time (BX=0) L1 objects (jets, tau, muons, EGamma).
+SumEt objects (MET,MHT,ET,HT) are defined with the PtPhiPairClass defined above.
+"""
+
+import ROOT
+
+from PhysicsTools.Heppy.analyzers.core.Analyzer import Analyzer
+from PhysicsTools.Heppy.analyzers.core.AutoHandle import AutoHandle
+from PhysicsTools.HeppyCore.utils.deltar import matchObjectCollection, matchObjectCollection3
+import PhysicsTools.HeppyCore.framework.config as cfg
+
+
+class PtPhiPairClass():
```

Comment:
Change name to PtPhiPair? 
(we're not appending the string Class at the end of class names)

Why not use a simple tuple with two values?